### PR TITLE
Add Katello docs for System Purpose with Bulk Actions

### PR DESCRIPTION
**Pull request by chris1984**

### CODE REVIEW

File: plugins/katello/nightly/user_guide/content_hosts/index.md [hunk 1]
```diff
@@ -172,4 +172,22 @@ To set System Purpose attributes for a host, Navigate to the Content Host Detail
 
 ![System Purpose](/plugins/katello/{{ page.version }}/user_guide/content_hosts/system_purpose.png)
 
+## Setting System Purpose attributes via Bulk Actions
 
+Select the Content Hosts that you wish to change the System Purpose attributes on.
+
+Next select the 'Bulk Actions' button in the top right, and choose 'Manage System Purpose'
+
+In the following modal, we can see at the top, how many Content Hosts will be affected by the change. Select the dropdown next to the attributes you want to change, select the desired value and when satisfied with the options, click the 'Assign' button.
```
Comment (jeremylenz) [inline plugins/katello/nightly/user_guide/content_hosts/index.md]: "Modal" is a UI term and users shouldn't be expected to know what it means.
```suggestion
In the following screen, a banner at the top displays how many Content Hosts will be affected by the change. Select the dropdown next to the attributes you want to change, select the desired value and when satisfied with the options, click the 'Assign' button.
```

File: plugins/katello/nightly/user_guide/content_hosts/index.md [hunk 2]
```diff
@@ -172,4 +172,22 @@ To set System Purpose attributes for a host, Navigate to the Content Host Detail
...
+
+![Bulk System Purpose](/plugins/katello/{{ page.version }}/user_guide/content_hosts/bulk_system_purpose.png)
+
+**Some following things to note about how the behaivor works with the dropdowns:**
+
+* **If 'No change' and 'None' are both selected, 'No change' wins**
+
+* **If both 'No change' and other addon items are selected, 'No change' wins**
+
+* **If both 'None (Clear)' and others are selected, None is ignored and the others win**
```
Comment (jeremylenz) [inline plugins/katello/nightly/user_guide/content_hosts/index.md]: I think there's no need to go into the details of how it behaves when the user selects illogical options such as both No change and None.  Rather, I think the most important things to highlight here are 

* If "No change" is selected, the value will remain what it was
* If "None (Clear)" is selected, any existing values will be erased

**Review by jeremylenz — CHANGES_REQUESTED** [review]
Thanks @chris1984 for keeping our docs up to date :)

I know this isn't part of the original change, but I think the "Set System Purpose attributes" section starting on line 169 is in the wrong place. It should be moved to before "Bulk Actions."

**Comment by chris1984** [comment]
@jeremylenz thanks for the review! updated based on your feedback.

File: plugins/katello/nightly/user_guide/content_hosts/index.md [hunk 3]
```diff
@@ -166,10 +172,20 @@ Next select the 'Bulk Actions' button in the top right.
...
+Select the Content Hosts that you wish to change the System Purpose attributes on.
 
-![System Purpose](/plugins/katello/{{ page.version }}/user_guide/content_hosts/system_purpose.png)
+Next select the 'Bulk Actions' button in the top right, and choose 'Manage System Purpose'
+
+In the following screen, a banner at the top displays how many Content Hosts will be affected by the change. Select the dropdown next to the attributes you want to change, select the desired value and when satisfied with the options, click the 'Assign' button.
+
+![Bulk System Purpose](/plugins/katello/{{ page.version }}/user_guide/content_hosts/bulk_system_purpose.png)
+
+**Some following things to note about how the behaivor works with the dropdowns:**
```
Comment (jeremylenz) [inline plugins/katello/nightly/user_guide/content_hosts/index.md]:
```suggestion
**Note**
```

Comment (jeremylenz) [reply plugins/katello/nightly/user_guide/content_hosts/index.md]: (I was going to call out the "behavior" typo but I think the single word is better)

**Comment by chris1984** [comment]
@jeremylenz updated and added to 3.18 section as well, since all of those are in the 3.18 branch at this point.

File: plugins/katello/nightly/user_guide/content_hosts/index.md (hunk 3 above)
Comment (chris1984) [reply plugins/katello/nightly/user_guide/content_hosts/index.md]: Updated.

File: plugins/katello/nightly/user_guide/content_hosts/index.md (hunk 2 above)
Comment (chris1984) [reply plugins/katello/nightly/user_guide/content_hosts/index.md]: Fixed

File: plugins/katello/nightly/user_guide/content_hosts/index.md (hunk 1 above)
Comment (chris1984) [reply plugins/katello/nightly/user_guide/content_hosts/index.md]: Fixed

**Review by jeremylenz — COMMENTED** [review]
Text changes look good :+1: 

For the screenshots:
* Probably better to select at least 2 systems so as not to highlight our bad grammar ("1 systems")
* Text size is too small; I would take it up 3 notches or so

**Comment by chris1984** [comment]
@jeremylenz updated

**Review by jeremylenz — APPROVED** [review]
thanks for updating the screenshots!

ACK

**Comment by chris1984** [comment]
@jturel can this get a merge please?

**Pull request merged**